src/rapids_metadata/__init__.py

```diff
@@ -242,6 +242,14 @@
 all_metadata.versions["25.06"].repositories["cugraph-gnn"].packages["libwholegraph"] = (
     RAPIDSPackage(has_wheel_package=True)
 )
+all_metadata.versions["25.06"].repositories["rapids-cli"] = RAPIDSRepository(
```

Comment:
Is "25.06" something that will continue changing when new rapids releases come out? 
I'm asking because if we will need to change this upon every release we should make a note somewhere or find a why of templating it somehow